packages/catalog-model/package.json

```diff
@@ -42,9 +44,11 @@
     "@types/express": "^4.17.6",
     "@types/jest": "^26.0.7",
     "@types/lodash": "^4.14.151",
+    "ajv-cli": "^4.0.1",
     "yaml": "^1.9.2"
   },
   "files": [
-    "dist"
+    "dist",
+    "schema"
```

Comment:
Would expect the compiled schema to be part of the dist build, or do you want the source schema to be part of the published package? It does increase the size a lil bit

Reply:
Yeah, I thought it would make sense to have the bare schemas in the package, which may me misguided :) Opinions welcome. The compiled version will be in dist anyway of course.

If I skip the whole pre-compile thing, I would move them into the source tree anyway and import them from ts code.